tests/unittests/OnnxImporterTest.cpp

```diff
@@ -252,12 +255,53 @@ TEST(onnx, importConv) {
   EE.compile(CompilationMode::Infer, F);
   EE.run(bindings);
   auto result = bindings.get(graphOutputVar)->getHandle();
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+  for (size_t i = 0; i < expectedDims.size(); i++)
```

Comment:
Use braces for for loop
You mean to loop over the expectedValues.size() right?

Reply:
Also same thing with `size_t e =  expectedValues.size()`

Reply:
Good catch. Done.